dbt/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "dbt"
+version = "0.0.1"
+readme = "README.md"
+requires-python = ">=3.11"
```

Comment:
I matched the required/used Python version across all our CI jobs and `pyproject` files to 3.11. Everyone _should_ be up to that as the default now on the server. If not, I can bump the global version.